server/chat-plugins/info.js

```diff
@@ -831,8 +845,12 @@ const commands = {
 
 		let targets = target.split(/[,+]/);
 		let sources = [];
-		let mod = Dex.mod(toId(targets[targets.length - 1])) || Dex;
-
+		let roomMod = Dex;
+		if (room && room.battle) {
+			let format = Dex.getFormat(room.battle.format);
+			roomMod = Dex.mod(format.mod);
+		}
+		let mod = Dex.mod(toId(targets[targets.length - 1])) || roomMod;
```

Comment:
You should support selection through a format as well as a mod name here too.

Reply:
OK, I made a change that should implement that. Does it look good now?